source/_components/azure.markdown

```diff
@@ -0,0 +1,66 @@
+---
+layout: page
+title: "Azure"
+description: "Instructions on how to setup the Azure component."
+date: 2018-08-28 10:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: microsoft_azure.png
+ha_category: Others
+featured: true
+ha_release: 0.77.1
+ha_iot_class: "Hub"
+---
+
+[Microsoft Azure](https://azure.microsoft.com) is fast becoming a popular choice as a cloud provider.
+ 
+The `azure` component is designed to let you interface your Home Assistant host with the Microsoft Azure cloud platform.
+
+To integrate a Microsoft Azure Subscription with Home Assistant, add the following section to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+azure:
+  client_id: 96b9XXXX-XXXX-XXXX-XXXX-XXXXcef0eff8
+  client_secret: SECRET-PASSWORD
+  tenant_id: 7912XXXX-XXXX-XXXX-XXXX-XXXX69bbace1
+  subscription_id: f231XXXX-XXXX-XXXX-XXXX-XXXXa3299dea
+
```

Comment:
Please remove this empty line since it wil show on the page

Reply:
done